kube-core/src/util.rs

```diff
@@ -52,4 +75,18 @@ mod test {
             Patch::Merge(()).content_type()
         );
     }
+
+    #[test]
+    fn cordon_patch_is_correct() {
+        use k8s_openapi::api::core::v1::Node;
+
+        let url = Node::url_path(&(), Some("ns"));
+        let req = Request::new(url).cordon("mynode").unwrap();
+        assert_eq!(req.uri(), "/api/v1/namespaces/ns/nodes/mynode?");
+        assert_eq!(req.method(), "PATCH");
+        assert_eq!(
+            req.headers().get("Content-Type").unwrap().to_str().unwrap(),
+            Patch::Merge(()).content_type()
```

Comment:
Relatively minor point, but his differs from what `kubectl cordon` is doing. Output from `k delete node1 -v=9`


```sh
Request Body: {"spec":{"unschedulable":true}}
curl -v -XPATCH  -H "Content-Type: application/strategic-merge-patch+json" -H "Accept: application/json, */*" -H "User-Agent: kubectl/v1.23.1 (linux/amd64) kubernetes/86ec240" 'https://0.0.0.0:39717/api/v1/nodes/node1'
```

shows a strategic merge patch instead. It probably doesn't make a difference here afaikt, but we should probably use the strategic merge strategy (which they have even promoted to the `kubectl patch` type default).